bedrock/firefox/templates/firefox/new/way-of-the-fox/scene1.html

```diff
@@ -0,0 +1,18 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/new/way-of-the-fox/base.html" %}
+
+{% block content_body %}
+<div class="main-content content">
+  {{ download_firefox(alt_copy=_('Free Download')) }}
+  <p class="tagline">
+    {{ _('<span>Quicken</span> your browsing. <span>Reclaim</span> your privacy. Enjoy your <span>freedom</span>.') }}
```

Comment:
I suspect this will only ever be seen in English, but if not, should add an l10n comment here about the spans.